src/api/app/jobs/measurements_job.rb

```diff
@@ -76,4 +77,10 @@ def notifications_measurements
       RabbitmqBus.send_to_bus('metrics', "notification_count,notifiable=#{type.first} value=#{type.second}")
     end
   end
+
+  def beta_features_measurements
+    ENABLED_FEATURE_TOGGLES.pluck(:name).each do |feature_name|
+      RabbitmqBus.send_to_bus('metrics', "beta_feature_count,feature=#{feature_name},action=disabled value=#{DisabledBetaFeature.where(name: feature_name).count}")
```

Comment:
Hm, why both? We can aggregate this in grafana no?

Reply:
True, I didn't consider this.

Reply:
I removed the callbacks now.